src/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "removeComments": false,
-    "noImplicitAny": false,
+    "noImplicitAny": true,
```

Comment:
Nice one - just read what this does and I like how it can be used to catch errors!